apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -243,6 +243,10 @@ class TopInstructions extends Component {
       RESIZER_HEIGHT;
 
     this.props.setInstructionsMaxHeightNeeded(maxNeededHeight);
+    // Force instruction area to resize to show full feedback area
+    if (this.state.tabSelected === TabType.COMMENTS) {
+      this.props.setInstructionsRenderedHeight(maxNeededHeight);
+    }
```

Comment:
why is this needed only for the comments/feedback area?

Reply:
this forces the area to resize to the full max needed height for the content. If you do this for something like instructions it can take over the whole screen which I'm guessing we did not want since that's not the way its been in the past.